packages/mermaid/src/diagrams/er/styles.js

```diff
@@ -33,6 +33,17 @@ const getStyles = (options) =>
     font-size: 18px;
     fill: ${options.textColor};
   }    
+  #MD_PARENT_START {
+    fill: #f5f5f5 !important;
+    stroke: ${options.lineColor} !important;
+    stroke-width: 1;
+  }
+  #MD_PARENT_END {
```

Comment:
This is not consistent with the other style selector naming conventions. Please use `.lowerCamelCase` class names.